doc-content/jbpm-docs/src/main/asciidoc/BPMN2/add-form-fields-proc.adoc

```diff
@@ -0,0 +1,302 @@
+[id='adding-form-fields']
```

Comment:
@michelehaglund I think this page is related to the old form modeller, it shouldn't be here

Reply:
Removed.